NeosModLoader/Properties/AssemblyInfo.cs

```diff
@@ -31,5 +31,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.10.1.0")]
-[assembly: AssemblyFileVersion("1.10.1.0")]
+[assembly: AssemblyVersion("1.11.0.0")]
+[assembly: AssemblyFileVersion("1.11.0.0")]
```

Comment:
The reason I haven't already done something like this is because I didn't think it was possible without doing some sort of build configuration nightmare thing. Am I missing something?

```cs
[assembly: AssemblyVersion(NeosModLoader.ModLoader.VERSION)]
[assembly: AssemblyFileVersion(NeosModLoader.ModLoader.VERSION)]
```

yields

```
An attribute argument must be a constant expression, typeof expression or array creation expression of an attribute parameter type
```

Reply:
Ah, cool, we have consts in C#. Yes, that would be a breaking change, and having a public const would be bad as it is a compile-time thing. e.g. if a mod dev were to compile against NML 1.9 but at runtime NML 1.10 is loaded, they'd still get a 1.9.

But I can just add a const in addition the the existing field to solve that problem.